test/e2e/coexistence/cli_coexistence_test.go

```diff
@@ -259,7 +259,7 @@ var _ = ginkgo.Describe("CLI Coexistence Tests", func() {
 				}
 				err := tf.PluginCmd.InstallPlugin(plugin.Name, target, plugin.Version)
 				gomega.Expect(err).To(gomega.BeNil(), "should not get any error for plugin install using new tanzu cli")
-				str, err := tf.PluginCmd.DescribePlugin(plugin.Name, plugin.Target)
+				str, err := tf.PluginCmd.DescribePluginLegacy(plugin.Name, plugin.Target)
```

Comment:
@chandrareddyp here we are running new tanzu cli so we should be using DescribePlugin